src/main/webapp/js/user-page-loader.js

```diff
@@ -40,8 +40,6 @@ function showMessageFormIfViewingSelf() {
       .then((loginStatus) => {
         if (loginStatus.isLoggedIn &&
             loginStatus.username == parameterUsername) {
```

Comment:
This code is deleted to make sure that message-form is hidden until the fetchBlobstoreUrlAndShowForm() has fetched the Blobstore url. that is why the code:
const messageForm = document.getElementById('message-form');
messageForm.classList.remove('hidden');
is in fact added inside the method fetchBlobstoreUrlAndShowForm(), after Blobstore url is fetched.